release/nightly_tests/dask_on_ray/dask_on_ray_sort_compute_template.yaml

```diff
@@ -1,12 +1,16 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 0
+aws:
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
 
-# TODO(ekl/sang) switch to i2.8xl and mount the NVMe disks
 head_node_type:
     name: head_node
     instance_type: i3.8xlarge
+    resources: {"object_store_memory": 51474836480}
```

Comment:
Shouldn't we use the defaults, or have ray set them correctly? Otherwise it's not testing what users see.

Reply:
Ah good point. It was set to avoid SIGBUS, but we don't need it anymore cuz fallback allocation is on.